src/Chartjs/composer.json

```
@@ -28,21 +28,20 @@
         }
     },
     "require": {
-        "php": ">=7.2.5",
+        "php": ">=8.0",
```

```suggestion
        "php": ">=8.1",
```

to be consistent with other recent Symfony packages